.github/workflows/test.yaml

```diff
@@ -74,11 +74,11 @@ jobs:
         run: |
           ./scripts/install_metallib.sh
 
-      - name: Add cluster credentials as a file if merlin/turing is being installed
+      - name: Add cluster credentials as a file if caraml/merlin/turing is being installed
         env:
           CHARTS_CHANGED: ${{ steps.list-changed.outputs.charts_changed }}
         run: |
-          if [[ $(echo $CHARTS_CHANGED | grep 'charts/\(merlin\|turing\|caraml\s\)') ]]; then
+          if [[ $(echo $CHARTS_CHANGED | grep 'charts/\(merlin\|turing\|caraml\)$') ]]; then
```

Comment:
The regex match change in https://github.com/caraml-dev/helm-charts/pull/378/files seems to have broken this step for `caraml`. Removed the `\s` and added a match for end of string instead.